make/family/ESP8266.make

```diff
@@ -66,6 +66,14 @@ CFLAGS+= -fno-builtin \
 -Wno-parentheses -Wno-type-limits -Wno-unused-function -Wno-unused-value \
 -Wl,EL -Wl,--gc-sections -nostdlib -mlongcalls -mtext-section-literals
 
+# only use mfore-l32 if 4MB board for now
+ifdef FLASH_4MB 
+MFORCE32 = `xtensa-lx106-elf-gcc --help=target | grep mforce-l32`
+ifneq ($(MFORCE32),)
+CFLAGS += -mforce-l32
```

Comment:
@MaBecker 

Does this need to be conditional?  Since the C code *relies* on MFORCE32 - if at this point the compiler does not have the flag you are up sh*t creek anyway!